apis/beacon/blocks/attestations.yaml

```diff
@@ -27,7 +30,9 @@ get:
               data:
                 type: array
                 items:
-                  $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Attestation'
+                  oneOf:
```

Comment:
```suggestion
                  anyOf:
```

Reply:
Not sure about this, IMU a block will contain only one type of attestations and the API should a list of one of them

Reply:
ah right this is an array, `oneOf` should be correct, it only caused issue in other apis and invalidated the openapi spec (https://github.com/ethereum/beacon-APIs/pull/421) but those were just single object cases

Reply:
I'm still confused about the usage of `oneOf` and `anyOf` for array items tbh, checked the spec and still not sure about it: In one hand there is the [`oneOf`](https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/#oneof) which states that the data should be valid against **one** of the specified schemas and in the other hand the [mixed typed array](https://swagger.io/docs/specification/data-models/data-types/#array) using `oneOf` which could contain both types

Reply:
Current approach seems correct to me, we need to apply the `oneOf` / `anyOf` on the array itself, not the items. Confirmed this results in the desired behavior with a [simple example](https://gist.github.com/nflaig/cd1cf5a03f9da6bcb055d639ddc8a6dd) using the schema validator we use.

Reply:
Great thanks! Used `anyOf` everywhere now